### DISCUSSION TITLE
简报：哔哩哔哩动态 ID 突破 800000000000000000

### DISCUSSION THREAD

**Discussion by Tinhone** [the1812/Bilibili-Evolved#4223] [综合讨论 / General]
昨晚 20:08 前后，哔哩哔哩动态 ID 突破 800000000000000000

注：800000000000000000，后面跟着 17 个零，是八十万兆

下面是废了点劲从一堆批量生成视频里找到的动态，可见信息产生速度是有多么恐怖：
https://t.bilibili.com/799999949040255075
https://t.bilibili.com/800000030651973673

**Comment by the1812** [comment]
这居然是递增的数字，我以为是类似bv号那样随机的
